### PR TITLE
Fix/rename dashboard components

### PR DESCRIPTION
**Pull request by taoteg**
## Overview: ##

- All refactored name updates and reduction in code complexity. 
- No visible changes to the app.
- All tests are passing.
- All linting is passing.

## Related Jira tickets: ##

* None

## Testing Steps: ##

1. Run project succesfully.

### CODE REVIEW

**Comment by taoteg** [comment]
Recommend this get merged into main LAST.

1) 'Update Analytics Dashboard' PR
2) 'Cleanup of ProTx subduir for project' PR
3) This PR 

** May require me running a FF locally off main (after 1 & 2 are merged) and resolving some deltas before this one merges cleanly, but I could be wrong.

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 1]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+              </>
+            );
+          }}
+        />
+        <Route
+          path={`${protxRoute}/demographics`}
+          render={() => {
+            // observedFeatures (i.e. Demographic Features) only has 2019 data.
+            setYear('2019');
+            setGeography('county');
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:117]:
```suggestion
```

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 2]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+              <>
+                <DisplaySelectors
+                  mapType={mapType}
+                  geography={geography}
+                  maltreatmentTypes={maltreatmentTypes}
+                  observedFeature={observedFeature}
+                  year={year}
+                  setMaltreatmentTypes={setMaltreatmentTypes}
+                  setObservedFeature={setObservedFeature}
+                  // limitToTopObservedFeatureFields
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:129]:
```suggestion
```

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 3]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+                </div>
+              </>
+            );
+          }}
+        />
+        <Route
+          path={`${protxRoute}/analytics`}
+          render={() => {
+            setMapType('predictiveFeatures');
+            setYear('2019');
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:167]:
```suggestion
```

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 4]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+                  </div>
+                </div>
+              </>
+            );
+          }}
+        />
+        <Route
+          path={`${protxRoute}/analytics`}
+          render={() => {
+            setMapType('predictiveFeatures');
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:166]:
```suggestion
            setMapType('observedFeatures');
```

This is untested but I'm guessing this is the issue where I am seeing wrong values displayed on map.

**Review by nathanfranklin — CHANGES_REQUESTED** [review]
The renaming of charts looks good 👍   Its nice to have this refactor and have improved names for components that match up better with how the researchers/end-users are referring to them.

Regarding `DashboardDisplay.js`, there is some issues in the refactor. I put some code suggestions in it and reaching out on slack.


-------

Not related to this PR but feels like DisplaySelectors is the next candidate for house keeping as its the victim of lots of last minute changes.

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 5]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+            return (
+              <>
+                <DisplaySelectors
+                  mapType={mapType}
+                  geography={geography}
+                  maltreatmentTypes={maltreatmentTypes}
+                  observedFeature={observedFeature}
+                  year={year}
+                  setMaltreatmentTypes={setMaltreatmentTypes}
+                  setObservedFeature={setObservedFeature}
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:128]:
```suggestion
                  setObservedFeature={setObservedFeature}
                  setGeography={setGeography}
```

File: client/src/components/ProTx/components/dashboard/DashboardDisplay.js [hunk 6]
```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
...
+            setMapType('observedFeatures');
+            return (
+              <>
+                <DisplaySelectors
+                  mapType={mapType}
+                  geography={geography}
+                  maltreatmentTypes={maltreatmentTypes}
+                  observedFeature={observedFeature}
+                  year={year}
+                  setMaltreatmentTypes={setMaltreatmentTypes}
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/dashboard/DashboardDisplay.js:127]:
```suggestion
```

**Review by nathanfranklin — APPROVED** [review]

**Comment by nathanfranklin** [comment]
LGTM 👍 

-----
There is a one other suggestion that has no effect on anything https://github.com/TACC/protx/pull/52#discussion_r700606205 but could be committed.  It really reflects that the DisplaySelectors is the next candidate for house keeping.

**Pull request merged**